### DISCUSSION TITLE
Label Text Auto Font Size

### DISCUSSION THREAD

**Discussion by tpitman** [xamarin/XamarinCommunityToolkit#1264] [Q&A] [answered]
I would like to have the font size on labels automatically adjust so that the text will fit in the space given to the label.

I found a reference to the effects:LabelSizeFontToFit, but don't see that in the current nuget package for this toolkit.

Is there another way to do this now?

**Comment by Cfun1** [comment, marked as answer]
There is a feature request and a pr related to what you are describing on this repo https://github.com/xamarin/XamarinCommunityToolkit/pull/1164

**Reply by tpitman** [reply]
Thank you. I guess I will try to find another solution until this one becomes available.
